.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,12 +5,8 @@
 
 #### :heavy_check_mark: Checklist
 
-<!--- Put an `x` in all the boxes that apply: -->
+<!--- Please include the following in your Pull Request when applicable: -->
 
-- [ ] All tests are passing `yarn test`
-- [ ] Screenshots attached (for UI changes)
```

Comment:
Hmmmm, I'm not sure I think this is clear enough. Would like to keep it.

Reply:
Added it back in

Reply:
Thanks!